app/components/concept/index.ts

```
@@ -186,6 +187,12 @@ export default class ConceptComponent extends Component<Signature> {
       }
     });
   }
+
+  @action
+  handleKeyDown() {
+    console.log("handleKeyDown"); 
+    this.isKeyboardNavigating = true;
+  }
```

ESLint

[error] Member handleKeyDown should be declared before member updateLastRevealedBlockGroupIndex.

(@typescript-eslint/member-ordering)